gammapy/data/pointing.py

```diff
@@ -30,31 +30,31 @@ class PointingMode(Enum):
 
     For ground-based instruments, the most common options will be:
     * POINTING: The telescope observes a fixed position in the ICRS frame
-    * DRIFT: The telscope observes a fixed position in the AltAz frame
+    * DRIFT: The telescope observes a fixed position in the AltAz frame
 
     OGIP also defines RASTER, SLEW and SCAN. These cannot be treated using
     a fixed pointing position in either frame, so they would require the
     pointing table, which is at the moment not supported by gammapy.
 
-    The H.E.S.S. data releases uses the not-defined value "WOBBLE",
-    which we assume to be the same as "POINTING", making the assumption
-    that one observation only contains a single wobble position.
+    Data releases based on gadf v0.2 do not have consistent OBS_MODE keyword
+    e.g. the H.E.S.S. data releases uses the not-defined value "WOBBLE".
+    For all gadf v0.2 data, we assume OBS_MODE to be the same as "POINTING",
+    making the assumption that one observation only contains a single wobble position.
```

Comment:
Maybe add that this is because gammapy anyways only supports a single fixed pointing position over the whole observation (either in equatorial or horizontal coordinates) at this moment?